src/lib/containers/EntityBadgeIcons.tsx

```diff
@@ -68,6 +70,8 @@ export type EntityBadgeIconsProps = {
   onUnlinkError?: (error: Error) => void
   /** Whether or not the badges (e.g. Annotations) can trigger opening a modal on click */
   canOpenModal: boolean
+  /** Whether this component should render a ReactTooltip or if an external component is managing it */
+  renderTooltipComponent: boolean
```

Comment:
For whatever reason, the Tooltip component struggles to set/update tooltip positioning when it's rendered within a `react-window`. It works better when we render the tooltip component outside of the virtualized list, so we make that an option for this component.